[2025-07-26T09:02:28.006+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-26T09:02:28.985+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_local_postgres_pipeline.load_artist_dim_table scheduled__2025-07-25T17:00:00+00:00 [queued]>
[2025-07-26T09:02:29.255+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_local_postgres_pipeline.load_artist_dim_table scheduled__2025-07-25T17:00:00+00:00 [queued]>
[2025-07-26T09:02:29.256+0700] {taskinstance.py:2306} INFO - Starting attempt 4 of 5
[2025-07-26T09:02:29.598+0700] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_artist_dim_table> on 2025-07-25 17:00:00+00:00
[2025-07-26T09:02:29.605+0700] {logging_mixin.py:188} WARNING - /home/ngtph/airflow/airflow_venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=6648) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-26T09:02:29.606+0700] {standard_task_runner.py:63} INFO - Started process 6656 to run task
[2025-07-26T09:02:29.605+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_local_postgres_pipeline', 'load_artist_dim_table', 'scheduled__2025-07-25T17:00:00+00:00', '--job-id', '546', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline_pjt.py', '--cfg-path', '/tmp/tmp02sb5uxr']
[2025-07-26T09:02:29.607+0700] {standard_task_runner.py:91} INFO - Job 546: Subtask load_artist_dim_table
[2025-07-26T09:02:32.742+0700] {task_command.py:426} INFO - Running <TaskInstance: etl_local_postgres_pipeline.load_artist_dim_table scheduled__2025-07-25T17:00:00+00:00 [running]> on host DESKTOP-9M5T9P5.localdomain
[2025-07-26T09:02:37.100+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='phuongnt226' AIRFLOW_CTX_DAG_ID='etl_local_postgres_pipeline' AIRFLOW_CTX_TASK_ID='load_artist_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2025-07-25T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-25T17:00:00+00:00'
[2025-07-26T09:02:37.103+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-26T09:02:37.915+0700] {base.py:84} INFO - Using connection ID 'postgres_id' for task execution.
[2025-07-26T09:02:38.656+0700] {sql.py:470} INFO - Running statement: 
    INSERT INTO artists (artistid, name, location, latitude, longitude)
    SELECT DISTINCT artist_id, artist_name, artist_location, artist_latitude, artist_longitude
    FROM staging_songs
    ON CONFLICT (artistid) DO NOTHING;
, parameters: None
[2025-07-26T09:02:38.765+0700] {sql.py:479} INFO - Rows affected: 0
[2025-07-26T09:02:38.818+0700] {logging_mixin.py:188} INFO - Executed SQL: 
    INSERT INTO artists (artistid, name, location... -> Result: None
[2025-07-26T09:02:38.819+0700] {python.py:237} INFO - Done. Returned value was: None
[2025-07-26T09:02:38.819+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-26T09:02:39.530+0700] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_local_postgres_pipeline, task_id=load_artist_dim_table, run_id=scheduled__2025-07-25T17:00:00+00:00, execution_date=20250725T170000, start_date=20250726T020228, end_date=20250726T020239
[2025-07-26T09:02:39.841+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-26T09:02:40.856+0700] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-26T09:02:40.959+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
